stages/org.osbuild.chown

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/python3
+"""
+Change file owner and group
+Change the file user and/or group ownership of one or more files or directories inside the tree.
+
+Notes:
+  - Requires 'chroot' in the buildroot.
+"""
+
+import os
+import subprocess
+import sys
+
+import osbuild.api
+from osbuild.util.path import in_tree
+
+SCHEMA_2 = r"""
+"options": {
+  "additionalProperties": false,
+  "properties": {
+    "items": {
+      "type": "object",
+      "additionalProperties": false,
+      "patternProperties": {
+        "^\\/(?!\\.\\.)((?!\\/\\.\\.\\/).)+$": {
+          "type": "object",
+          "required": ["paths"],
+          "anyOf": [
+            {"required" : ["user", "uid"]},
+            {"required" : ["group", "gid"]}
+          ],
+          "properties": {
+            "uid": {
+              "description": "User UID",
+              "type": "number"
+            },
+            "gid": {
+              "description": "User GID",
+              "type": "number"
+            },
+            "user": {
+              "type": "string",
+              "description": "User name"
+            },
+            "group": {
+              "type": "string",
+              "description": "Group name"
+            },
```

Comment:
Thinking about this more, I think that there should be just one property per user and group and the schema should just allow specifying it as a string (name) or as a number (ID).

In addition, the string and number types should specify further constraints. IOW not any string (e.g. https://github.com/osbuild/osbuild/blob/b97b78382f4d9edab1f4ec6b1507a9a6b5edc0c7/stages/org.osbuild.users#L29 and https://github.com/osbuild/osbuild/blob/b97b78382f4d9edab1f4ec6b1507a9a6b5edc0c7/stages/org.osbuild.groups#L27) or number are valid values (can't be negative).

Nevertheless `chown` would fail in case of invalid values, so this additional validation is just a plus, not a must.